moped-editor/src/components/GridTable/Filters.js

```diff
@@ -367,12 +385,16 @@ const Filters = ({
   /**
    * Clears the filters
    */
-  const handleClearFilters = () => {
+  const handleClearFilters = useCallback(() => {
```

Comment:
i dont think this needs a `useCallback` anymore, in the past this function was called within a `useEffect` but now its not